typing/printtyp.mli

```diff
@@ -49,6 +49,13 @@ val wrap_printing_env: error:bool -> Env.t -> (unit -> 'a) -> 'a
     (* This affects all the printing functions below *)
     (* Also, if [~error:true], then disable the loading of cmis *)
 
+(** [wrap_printing_env_error env f] does something else in addition to
+    [wrap_printing_env ~error:true env f]: it also wraps all functions
+    in the returned [Location.error] value to disable the loading of cmis.
+*)
```

Comment:
`something else` is a bit unclear. I would propose
```
[wrap_printing_env_error env f] ensures that all printing functions in a `Location.error` report
are evaluated within the [wrap_printing_env ~error:true env] context
```

Reply:
I adopted your suggestion with the additional sentence `(The original call to [f] is also evaluated within that context.)`.